src/main.js

```diff
@@ -1,17 +1,25 @@
 import { ValidatorPollingService } from './validator-polling-service.js';
-import { SMSNotifier } from './notifiers/index.js';
+import { SMSNotifier, TelegramNotifier } from './notifiers/index.js';
 import { config } from './config.js';
 import { BeaconAPIClient } from './beacon-api-client.js';
 import { validatorBalanceReducedAlert, validatorStatusChangedAlert } from './alerts/index.js';
 import { logger } from './logger.js';
 
-const smsNotifer = new SMSNotifier(config.sms)
+const notifiers = [];
+if (config.sms.accountSid) notifiers.push(new SMSNotifier(config.sms));
```

Comment:
`config.sms` and `config.telegram` are truthy, so need to access something inside. 
Maybe we should consider adding a flag like `config.sms.enabled`?

Reply:
`enabled` is not going to be useful if the token is missing. I think ideally the constructor throws an error if any of the parameters are missing, or perhaps we could send an initial message to say that "the notifier has been setup", that way we know it's properly been set up; can do this in a separate PR though. What do you think?